server/src/test/java/org/elasticsearch/discovery/SnapshotDisruptionIT.java

```diff
@@ -135,9 +135,14 @@ public void clusterChanged(ClusterChangedEvent event) {
         assertBusy(() -> {
             SnapshotsInProgress snapshots = dataNodeClient().admin().cluster().prepareState().setLocal(false).get().getState()
                 .custom(SnapshotsInProgress.TYPE);
+            SnapshotDeletionsInProgress snapshotDeletionsInProgress =
+                dataNodeClient().admin().cluster().prepareState().setLocal(false).get().getState().custom(SnapshotDeletionsInProgress.TYPE);
```

Comment:
`local = false` is the default. No need to specify this here (also remove from one line above). Perhaps only fetch the cluster state once